hack/validate-canary.sh

```diff
@@ -107,28 +106,32 @@ spec:
   containers:
   - name: oci-cloud-controller-manager-canary-test-runner
     image: iad.ocir.io/oracle/oci-cloud-controller-manager-canary:${version}
-    command: ["/bin/bash"]
-    args: ["-ec", "make canary"]
     env:
-      - name: METRICS_FILE
-        value: /metrics/output.json
-      - name: KUBECONFIG_VAR
-        value: $(cat ${KUBECONFIG} | openssl enc -base64 -A)
+    - name: KUBECONFIG_VAR
+      value: $(cat ${KUBECONFIG} | openssl enc -base64 -A)
+    - name: METRICS_FILE
+      value: /metrics/output.json
+    - name: MONITOR_PERIOD
+      value: "30"
+    - name: CANARY_MODE
+      value: monitor
+    command: ["/bin/bash"]
+    args: ["-ec", "/oci/scripts/ccm-canary-entrypoint.sh"]
     volumeMounts:
     - mountPath: /metrics
       name: metrics-volume
- 
+
   - name: oci-cloud-controller-manager-canary-test-reporter
     image: iad.ocir.io/oracle/oci-cloud-controller-manager-ci-e2e:1.0.1
     command: ["/bin/bash"]
-    args: ["-ec", "touch \$METRICS_FILE; while [ -z \$(cat \$METRICS_FILE | grep 'end_time' | cut -d':' -f 1) ]; do sleep 1;  done; cat \$METRICS_FILE"]
+    args: ["-ec", "while true; do sleep 10; cat \$METRICS_FILE; done"]
```

Comment:
The canary validation tests now has a dumb reporter that just sits in an infinite loop and cats the contents of the metrics file to std out.